### PR TITLE
DOCSP-37363 - What's New

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-37363
Staging:
- [What's New](https://preview-mongodbmongokart.gatsbyjs.io/pymongo/docsp-36146-whats-new/whats-new/)
- [Breaking Changes](https://preview-mongodbmongokart.gatsbyjs.io/pymongo/docsp-36146-whats-new/upgrade/#std-label-version-4.7-breaking-changes)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/upgrade.txt
Comment (mongoKart) [inline source/upgrade.txt:1]: The rest of this page is outside the scope of this ticket; I just needed the section to link to from What's New.

**Review by rachel-mack — APPROVED** [review]
LGTM
